Wikipedia/Code/ReadingListDetailViewController.swift

```diff
@@ -548,6 +569,14 @@ extension ReadingListDetailViewController: ReadingListDetailExtendedViewControll
     func extendedViewController(_ extendedViewController: ReadingListDetailExtendedViewController, didBeginEditing textField: UITextField) {
         editController.isTextEditing = true
     }
+    
+    func extendedViewController(_ extendedViewController: ReadingListDetailExtendedViewController, titleTextFieldWillClear textField: UITextField) {
+        navigationItem.rightBarButtonItem?.isEnabled = false
+    }
+    
+    func extendedViewController(_ extendedViewController: ReadingListDetailExtendedViewController, titleTextFieldWillChangeCharacters textField: UITextField, replacementString string: String) {
+        navigationItem.rightBarButtonItem?.isEnabled = textField.text?.count == 1 && string.isEmpty ? false : true
```

Comment:
Do you recall if `isEmpty` accounts for whitespace? If not could we use `!string.wmf_hasNonWhitespaceText` instead of `isEmpty` here so it considers the field empty if the user enters just a bunch of spaces?

Reply:
👍 fix pushed